lib/iris/cube.py

```diff
@@ -2205,12 +2205,11 @@ def new_cell_measure_dims(cm_):
             first_slice = next(slice_gen)
         except StopIteration:
             first_slice = None
-        if self._numpy_array is not None:
```

Comment:
We've defined a method to make this check, so let's use it rather than repeating the same code.